properties.xml

```diff
@@ -24,7 +24,7 @@
   <version>v. @repository.version@ for MEI 4.0.0</version>
   <footer>
     <div class="footer" xmlns="http://www.w3.org/1999/xhtml">
-      <div style="float:right" class="version">v. @repository.version@ for MEI 4.0.0</div>
+      <div style="float:right" class="version">v. {$config:version} for MEI 4.0.0</div>
```

Comment:
Haven't tested yet because of the other bug, but does this work with the "templating"?

Reply:
Fixed the endless loop and tried to consolidate property replacement.